lsp-metals.el

```diff
@@ -574,6 +604,19 @@ WORKSPACE is the workspace we received notification from."
                             'local-map (lsp-metals--status-string-keymap workspace command?))
       workspace)))
 
+(lsp-defun lsp-metals--quick-pick (_workspace (&MetalsQuickPickParams :items :place-holder?))
+  "The Metals quick pick request is sent from the server to the client to let the user provide a string value by picking one out of a number of given options."
```

Comment:
Could you please split these long descriptions into lines? First sentence shouldn't be longer 80 characters.

Reply:
Of course, I shortened the descriptions.

Reply:
It could be just split to sentences and convert to multiline, but that's fine :)